Frends.MicrosoftSQL.ExecuteProcedure/README.md

```diff
@@ -0,0 +1,30 @@
+# Frends.MicrosoftSQL.ExecuteProcedure
+Frends MicrosoftSQL Task to execute MSSQL procedure.
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.MicrosoftSQL/actions/workflows/ExecuteProcedure_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.MicrosoftSQL/actions)
+![MyGet](https://img.shields.io/myget/frends-tasks/v/Frends.MicrosoftSQL.ExecuteProcedure)
+![Coverage](https://app-github-custom-badges.azurewebsites.net/Badge?key=FrendsPlatform/Frends.MicrosoftSQL/Frends.MicrosoftSQL.ExecuteProcedure|main)
+
+# Installing
+
+You can install the Task via Frends UI Task View or you can find the NuGet package from the following NuGet feed https://www.myget.org/F/frends-tasks/api/v2.
```

Comment:
Let's drop the myget URL completely. Just `You can install the Task via Frends UI Task View` should be fine from now on.